baas-smoke-tests/src/test/resources/kubernetes/interactions-example-update-config-map.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: reserve-items
+  labels:
+    custom-resource-definition: interactions
+data:
+  image: baas-interaction-example-reserve-items:1.0.0
```

Comment:
how/where are `imagePullSecrets` specified?

Reply:
Secrets can be either mounted or added to environment variables, and taken from the Kubernetes secrets store. The `interactions-example-config-map.yaml` has some secrets configured as example